ssl/record/rec_layer_s3.c

```diff
@@ -869,27 +871,31 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
             SSL3_RECORD_add_length(thiswr, 1);
 
             /* Add TLS1.3 padding */
+            rlen = SSL3_RECORD_get_length(thiswr);
+            max_padding = SSL3_RT_MAX_PLAIN_LENGTH - rlen;
```

Comment:
It is explicitly disallowed:

`The presence of padding does not change the overall record size limitations - the full encoded TLSInnerPlaintext MUST not exceed 2^14 octets. If the maximum fragment length is reduced, such as by the max_fragment_length extension from [RFC6066], then the reduced limit applies to the full plaintext, including the padding.`